### PR TITLE
Server side form validation

### PR DESCRIPTION
**Pull request by JoshuaMitchell**

### CODE REVIEW

File: app/main/forms/step_0.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+from wtforms import StringField, SelectField
+from wtforms.validators import DataRequired, Email, Regexp
+from flask_babel import gettext
+
+counties = ["Allen","Anderson","Atchison","Barber","Barton","Bourbon","Brown","Butler","Chase","Chautauqua","Cherokee","Cheyenne","Clark","Clay","Cloud","Coffey","Comanche","Cowley","Crawford","Decatur","Dickinson","Doniphan","Douglas","Edwards","Elk","Ellis","Ellsworth","Finney","Ford","Franklin","Geary","Gove","Graham","Grant","Gray","Greeley","Greenwood","Hamilton","Harper","Harvey","Haskell","Hodgeman","Jackson","Jefferson","Jewell","Johnson","Kearny","Kingman","Kiowa","Labette","Lane","Leavenworth","Lincoln","Linn","Logan","Lyon","Marion","Marshall","McPherson","Meade","Miami","Mitchell","Montgomery","Morris","Morton","Nemaha","Neosho","Ness","Norton","Osage","Osborne","Ottawa","Pawnee","Phillips","Pottawatomie","Pratt","Rawlins","Reno","Republic","Rice","Riley","Rooks","Rush","Russell","Saline","Scott","Sedgwick","Seward","Shawnee","Sheridan","Sherman","Smith","Stafford","Stanton","Stevens","Sumner","Thomas","Trego","Wabaunsee","Wallace","Washington","Wichita","Wilson","Woodson","Wyandotte"]
+class FormStep0(FlaskForm):
```
Comment (karpet) [inline app/main/forms/step_0.py]: seems like this fixture of array-of-counties is going to be duplicated elsewhere? could we define a class or some singleton or global var that contains this list for re-use?

File: requirements.txt [hunk 2]
```diff
@@ -15,7 +15,6 @@ Flask-WTF==0.14.2
 idna==2.6
 itsdangerous==0.24
 Jinja2==2.10
-ksmyvoteinfo==0.2
```
Comment (karpet) [inline requirements.txt]: seems like this is a bad merge?

File: app/templates/index.html [hunk 3]
```diff
@@ -146,28 +44,28 @@ <h1 class="h1-responsive">{{_('0_intro')}}</h1>
...
-  // form.submit(function( event ){
-  //   event.preventDefault();
-  //   var formy = new Formy(form);
-  //   formy.submitForm();
-  //
-  // })
-});
-
-</script>
+// $(function () {
```
Comment (karpet) [inline app/templates/index.html]: commented out because no longer needed? delete instead? or just unused for now?

File: requirements.txt (hunk 2 above)
Comment (JoshuaMitchell) [reply requirements.txt]: Yes will fix.

File: app/templates/index.html (hunk 3 above)
Comment (JoshuaMitchell) [reply app/templates/index.html]: just commented out for now.  testing the server side validation before i add back in the js validation.

**Pull request merged**